exporter/test/e2e/tomcat_test.go

```diff
@@ -62,3 +62,32 @@ func TestJBossWebServer(t *testing.T) {
 		}))
 	_ = testenv.Test(t, feature.Feature())
 }
+
+func TestJBossWebServer_6_0_5(t *testing.T) {
+
+	appName := "jboss-webserver-6-0-5"
+	containerName := "main"
+	image := "registry.redhat.io/jboss-webserver-6/jws60-openjdk21-openshift-rhel8@sha256:d038212f1d7e72440929dd014b13611e761d6149939789d0b18d5b8d3af1a9fb"
+	deployment := newAppDeployment(namespace, appName, 1, containerName, image)
+        
+	// JWS 6.0.5 does not include Tomcat redhat build version in MANIFEST file
+	// https://issues.redhat.com/browse/JWS-3528
+	feature := features.New("JBoss WebServer 6.0.5 from base image "+image).
+		Setup(deployTestResource(deployment, appName)).
+		Teardown(undeployTestResource(deployment, appName)).
+		Assess("runtime info extracted", checkExtractedRuntimeInfo(namespace, "app="+appName, containerName, func(g *Ω.WithT, runtimeInfo types.ContainerRuntimeInfo) {
+			expected := types.ContainerRuntimeInfo{
+				Os:              "rhel",
+				OsVersion:       "8.10",
+				Kind:            "Java",
+				KindVersion:     "21.0.6",
+				KindImplementer: "Red Hat, Inc.",
+				Runtimes: []types.RuntimeComponent{{
+					Name:    "Apache Tomcat",
+					Version: "10.1.8.redhat-00022",
```

Comment:
@liborfuka it is what it is, the JBoss WebServer 6.0.5 that is shipped reports a tomcat version of `10.1.8`.
Our tests should just use this version of the assertion

Reply:
ok, assertion updated.